### PR TITLE
Remove necessery registerElements

### PR DESCRIPTION
**Pull request by korneliakobiela**
Remove necessery registerElements

Signed-off-by: Kornelia Kobiela <k.kobiela@samsung.com>

### CODE REVIEW

**Comment by korneliakobiela** [comment]
CI doesn't work because JSDOM doesn't have implemented Custom Elements v1.

File: design-editor/src/pane/design-editor-element.js [hunk 1]
```diff
@@ -372,27 +372,27 @@ class DesignEditor extends DressElement {
...
-		this._selectLayer = new SelectLayerElement();
+		this._selectLayer = new SelectLayer();
 		this._selectLayer.options = {
 			designEditor: this,
 			screenWidth: this.screenConfig.width,
 			screenHeight: this.screenConfig.height,
 			screenRatio: this.screenConfig.ratio
 		};
-		this._sectionController = new SectionControllerElement();
+		//this._sectionController = new SectionController();
```
Comment (hsiwkin) [inline design-editor/src/pane/design-editor-element.js]: Please remove commented code

File: design-editor/src/pane/design-editor-element.js [hunk 2]
```diff
@@ -372,27 +372,27 @@ class DesignEditor extends DressElement {
...
+		this._tooltipPanel = new Tooltip();
 
 		snapGuides = SnapGuideManager.getInstance().getGuides();
 		this._$scroller
 			.append(this._$iframe)
 			.append(this._$iframeDummy)
-			.append(this._selectLayer)
-			.append(this._sectionController);
+			.append(this._selectLayer);
+		//.append(this._sectionController);
```
Comment (hsiwkin) [inline design-editor/src/pane/design-editor-element.js]: as above : )

File: design-editor/src/system/stage-manager.js [hunk 3]
```diff
@@ -7,14 +7,14 @@ import {EVENTS, eventEmitter} from '../events-emitter';
...
-import {StructureElement} from '../panel/property/structure/structure-element';
-import HTMLAssistant from '../pane/select-layer/html-assistant';
-import {ToolbarElement} from '../panel/toolbar-element';
-import {PreviewElement} from '../panel/preview/preview-element';
-import {PreviewToolbarElement} from '../panel/preview/preview-toolbar-element';
-import {InfoElement} from '../pane/select-layer/info-element';
+import {PropertyContainer} from '../panel/property/property-container-element';
+//import {AnimationContainerElement} from '../panel/animation/animation-container-element';
+import {Structure} from '../panel/property/structure/structure-element';
+//import HTMLAssistant from '../pane/select-layer/html-assistant';
```
Comment (hsiwkin) [inline design-editor/src/system/stage-manager.js]: remove commented code : )

**Pull request merged**